### ISSUE TITLE
Normal maps not visible on refract material

### ISSUE THREAD

**Issue by Vigilabo** [Frooxius/NeosPublic#1639]
## Describe the bug
Normal maps do not work on the refract material

## To Reproduce
1. Put a normal map on the refract material

## Expected behavior
The normal map should affect the material

## Bug information:
 - How often does it happen: Always
 - Does the bug persist after restarting Neos? Yes
 - Neos Version: Beta 2021.1.22.1352
 - Neos Platform: Windows
 - Regression: Yes, it was functional on the previous build

## Reporters:
Vigilabo

**Comment by Frooxius** [comment]
I can't replicate this, Normal maps work correctly on Refract materials on my end. Did you wait a bit? It might've been that the variant hasn't been computed yet.

**Comment by Hayden-da-Serg** [comment]
In the case that prompted this, there were three users present in the session, me, @Coffee-Maker and @Vigilabo, and all three of us saw the same thing, the normal map was not getting applied to the refraction at all.

**Comment by Frooxius** [comment]
That seems consistent with the variant of the shader being generated on the cloud. Did you wait a bit and see if it loads? Does the issue still persist?

**Comment by dotuser** [comment]
Not specifically related to refraction maps, but was recommended by Shifty to post here. Here's a pure copy of the text from Discord:
"Probably not a bug but since the asset variant system has been fully implemented it now takes upwards of 8+ minutes for a 4k normal to fully load (freshly loaded into a world without anyone in it but me, from computer) when it used to only take maybe 2. This is kind of annoying."

**Comment by Frooxius** [comment]
Is it the texture itself that's not loading or just not showing up on the material? Does this happen only once or every time? How many times have you tried to replicate this?

Nothing has really changed with the asset variant system for textures, so that should be the same. Once the texture is cached it should be loaded fully locally too.

I'm sorry for the annoyance, but unfortunately I'll need more information to diagnose this properly.

**Comment by dotuser** [comment]
It steps through each resolution variant from lowest to highest much slower than it used to. It *finishes*, but 8+ minutes later. Which isn't great when you're trying to make multiple changes and check that they look good.

**Comment by Frooxius** [comment]
This doesn't sound like it's related to this issue at all, it might be better to create a new one instead.

**Comment by Hayden-da-Serg** [comment]
I was checking up on one of my avatars, and the normal map on Refraction is just not working.
Here's the screenshots of what it looks like in-app and what the settings are. I sadly don't have any screenshots of what it's supposed to look like nor do I have a way to go back to a version in which this did function properly in.
![image](https://user-images.githubusercontent.com/25524760/105878243-0f4c0f80-5fbe-11eb-8f22-7d61d08f3acc.png)
![image](https://user-images.githubusercontent.com/25524760/105878282-17a44a80-5fbe-11eb-9669-db5b4fb2ad89.png)
*The effect is kind of hard to see, though there's supposed to be distortion on the refraction from the normal map*

**Comment by Ultranique** [comment]
I can verify this is not working for me as well, even on some previously working meshes that had the material applied - all are now broken and can not be restored from the same versions saved to inventory. I can see the normal map is in the correct slot, but it is ignored even after saving to inventory and spawning a new copy. I have remade the material multiple times in different worlds (owned by me as well as others), and can reproduce this each time. I have tried random normal maps just to see if it was a specific one, and nothing I do will create the previously working refractive effect with the normal applied.

**Comment by Ultranique** [comment]
The last time I saw the refractive material work correctly was on 1/14/2021 (give or take a few days):

![refractive_issue_20210114230915_1](https://user-images.githubusercontent.com/34196280/106163976-20785600-614f-11eb-8652-baaf85be3d15.jpg)

The eyes on this creature are showing the normal map correctly. I had saved 3 different versions of this avatar and when I came back to the world they are in (as well as items in my inventory), this material was no longer correct - all versions of it now show no normal distortions, but the texture is in the normal map slot. I can pull the texture out of the material, inspect it - everything seems normal, but the effect just does not show up any longer.
